constitution/article-06.tex

```diff
@@ -69,12 +70,15 @@
   \item Attend weekly RITSEC meetings.
 \end{enumerate}
 
-% SECTION E: COMPETITION ARCHITECT
+% SECTION E: COMPETITION ARCHITECTS
 \Section{Competition Architect}
 
 \begin{enumerate}
   \item This position is elected by the club members.
+  \item Two Competiton architects will be elected. Each after their major competition.
```

Comment:
Thoughts on identifying them via competition instead of Primary Vs Secondary.
ie. IRSEC CA & ISTS CA